dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/DefaultTrackerObjectsDeletionService.java

```diff
@@ -107,32 +96,16 @@ public TrackerTypeReport deleteEnrollments( TrackerBundle bundle, TrackerType tr
         {
             String uid = enrollments.get( idx ).getEnrollment();
 
-            TrackerObjectReport trackerObjectReport = new TrackerObjectReport( TrackerType.ENROLLMENT );
-
             ProgramInstance programInstance = programInstanceService.getProgramInstance( uid );
 
-            if ( bundle.getUser() != null )
-            {
-                // TODO authority check should be part of validation phase. This
-                // check will be moved in validation hooks.
-                List<TrackerErrorReport> trackerErrorReports = isAllowedToDeleteEnrollment( idx, bundle.getUser(),
-                    programInstance, bundle );
-
-                if ( !trackerErrorReports.isEmpty() )
-                {
-                    return addErrorToTypeReport( typeReport, trackerObjectReport, trackerErrorReports, idx, uid );
-                }
-            }
-
             List<Event> events = eventTrackerConverterService
                 .to( Lists.newArrayList( programInstance.getProgramStageInstances()
                     .stream().filter( psi -> !psi.isDeleted() )
                     .collect( Collectors.toList() ) ) );
 
-            TrackerBundle trackerBundle = TrackerBundle.builder().events( events ).user( bundle.getUser() ).build();
+            TrackerBundle trackerBundle = new TrackerBundle().toBuilder().events( events ).user( bundle.getUser() )
+                .build();
```

Comment:
Is there any difference in doing this rather than it was on the previous revision?